src/traces/parcoords/defaults.js

```diff
@@ -101,7 +97,15 @@ module.exports = function supplyDefaults(traceIn, traceOut, defaultColor, layout
         traceOut.visible = false;
     }
 
-    coerceFont('labelfont', coerce, layout.font, attributes.labelfont.dflt || {});
-    coerceFont('tickfont', coerce, layout.font, attributes.tickfont.dflt || {});
-    coerceFont('rangefont', coerce, layout.font, attributes.rangefont.dflt || {});
+    // make default font size 10px,
+    // scale linearly with global font size
+    var fontDflt = {
+        family: layout.font.family,
+        size: Math.round(layout.font.size * (10 / 12)),
```

Comment:
@monfera this is what I came off with. 

As you pointed out, changing the font size default to 12 makes most parcoord graphs look less-than-optimal. We should fix that text-overlapping problem at some point though - similar to how we do it currently in [cartesian](https://github.com/plotly/plotly.js/blob/2bac6ef08f46dae0117ba34dea1599866f060379/src/plots/cartesian/axes.js#L1844-L1897) axes. But that's for another PR. So, the 10px default value remain. 

But, to make this a little more plotly-esque, we make the parcoord font size scale linearly with `layout.font` similar to what's done for axes [`titlefont`](https://github.com/plotly/plotly.js/blob/2bac6ef08f46dae0117ba34dea1599866f060379/src/plots/cartesian/axis_defaults.js#L81-L85).